site/content/en/blog/releases/1.5.0-rc.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "1.5.0 - Release Candidate"
+linkTitle: "1.5.0-rc"
+date: "2020-04-06"
```

Comment:
04-07?

Reply:
I usually put it a day behind, just to make sure it publishes - just in case of weird timezone differences.